src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -189,15 +189,27 @@ protected function addCookieToResponse($request, $response)
         }
 
         $response->headers->setCookie(
-            new Cookie(
-                'XSRF-TOKEN', $request->session()->token(), $this->availableAt(60 * $config['lifetime']),
-                $config['path'], $config['domain'], $config['secure'], false, false, $config['same_site'] ?? null
-            )
+            $this->newCookie($request, $config)
         );
 
         return $response;
     }
 
+    /**
+     * Create the CSRF token that will be sent with the response.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Illuminate\Config\Repository
```

Comment:
$config is an array here, not a `\Illuminate\Config\Repository` instance.

Reply:
Thanks for catching that! Should be fixed.